launch.go

```diff
@@ -382,6 +382,9 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		"SD_JOB_NAME":            oldJobName,
 		"SD_PIPELINE_NAME":       pipeline.ScmRepo.Name,
 		"SD_PULL_REQUEST":        pr,
+		"SD_PR_PARENT_JOB_ID":    strconv.Itoa(job.PrParentJobID),
+		"SD_PARENT_BUILD_ID":     fmt.Sprintf("%v", build.ParentBuildID),
```

Comment:
why isn't this using `strconv.Itoa` as well?

Reply:
`ParentBuildID` is type interfact so I couldn't use `strconv.Itoa`: https://github.com/screwdriver-cd/launcher/blob/master/screwdriver/screwdriver.go#L136